source/layouts/blog.erb

```diff
@@ -18,23 +18,121 @@
+            <div class="social-icons__icon">
+              <%= image_tag 'twitter.png', class: 'social-icons__image', alt: 'Twitter' %>
+            </div>
+          </a>
+          <a class="social-icons__icon-container" href="https://www.facebook.com/pages/Unboxed-Consulting/114802528557320" target="_blank">
+            <div class="social-icons__icon">
+              <%= image_tag 'facebook.png', class: 'social-icons__image', alt: 'Facebook' %>
+            </div>
+          </a>
+          <a class="social-icons__icon-container" href="http://www.linkedin.com/company/unboxed-consulting" target="_blank">
+            <div class="social-icons__icon">
+              <%= image_tag 'linkedin.png', class: 'social-icons__image', alt: 'LinkedIn' %>
+            </div>
+          </a>
+        </div>
+      </section>
+
+      <section class="blog-author">
+        <div class="blog-author__container">
+          <div class="blog-author__image">
+            <img src="<%= data.people.graeme.image %>" alt="blog author"/>
+          </div>
+          <aside class="blog-author__right-section">
+            <p class="blog-author__name">
+              <span><%= data.people.graeme.name %></span>
+            </p>
+            <div class="blog-author__bio">
+              <%= data.people.graeme.bio %>
+            </div>
+          </aside>
+          <div class="blog-author__clear"></div>
+        </div>
+      </section>
 
-    <div class="page-content">
-      <%= yield %>
       <%= partial 'disqus', locals: { disqus_identifier: current_page.data.disqus_identifier } %>
-      <ul>
-        <% current_article.similar_articles.first(3).each do |article| %>
-          <li><%= article.title %></li>
-        <% end %>
-      </ul>
+
+      <% if current_article.similar_articles.any? %>
+        <section class="blog-related">
+          <p class="blog-related__header">Related</p>
+          <div class="blog-related__container">
+            <% current_article.similar_articles.first(3).each do |article| %>
+              <div class="blog-related__tile">
+                <% if current_page.data.tags %>
+                  <div class="blog-related__tag">
+                    <% current_page.data.tags.each do |tag| %>
+                      <div class="blog-related__tag-link"><a href=""><%= tag %></a></div>
+                    <% end %>
+                  </div>
+                <% end %>
+                <div class="blog-related__date" id="articlePublishedDate" data-date="<%= article.date %>">
+                <span>
+                  <%= Time.parse(article.date.to_s).strftime("%e %b %Y") %>
+                </span>
+                </div>
+
+                <script type="text/javascript">
+                  date = document.querySelector('#articlePublishedDate').getAttribute('data-date');
+                  document.querySelector('#articlePublishedDate span').outerHTML = moment(new Date(date)).fromNow();
```

Comment:
Yes, we'd need it as date format in related articles tiles is different. But it might be changed in the future as related articles part is still in progress in terms of design.